### ISSUE TITLE
zombie rancher processes

### ISSUE THREAD

**Issue by robertstagg** [rancher/rancher#10713]
How can I stop this:
ERROR [:] [] [] [] [ecutorService-6] [.e.s.i.ProcessInstanceDispatcherImpl] Unknown exception running process [service.remove:5852] on [12], canceled by [State [purged] is not valid for process [instance.stop:7859] on resource [144]]

The service was deleted and no longer exists. Do I need to delete records from MySQL or is there a sysadmin command to tell Rancher stop wasting your time on this?

---
| Useful | Info |
| :-- | :-- |
|Versions|Rancher `v1.6.12` Cattle: `v0.183.30` UI: `v1.6.30` |
|Access|`localauth` `admin`|
|Orchestration|`Cattle`|
|Route|`service.containers`|

**Comment by aemneina** [comment]
sounds like you have some inconsistency between your resources. an easy fix might be to update the offending resource and set it to 'stopped' possibly. in the processes tab in the ui, you should be able to see an instance.stop process. drill down into that to get the instance id and share the sanitized details for more help.

**Comment by robertstagg** [comment]
Easy fix won't work. The service no longer exists. 
In the process tab I have service.remove process. If I expand that I see that every 2 min it tries again. At one point I got a delete option on the process. when I clicked that I ended up with 2 service.remove tasks. Now I only get reply or API. The (deleted) service is namenode-primary in hadoop.
Image:
rancher/hadoop-base:v0.3.5
Command:
su -c sleep 15 && /usr/local/hadoop-2.7.1/bin/hdfs namenode hdfs
